internal/pkg/service/buffer/storage/compression/config.go

```diff
@@ -53,7 +55,7 @@ func DefaultGZIPConfig() Config {
 			Level:       DefaultGZIPLevel,
 			Impl:        DefaultGZIPImpl,
 			BlockSize:   DefaultGZIPBlockSize,
-			Concurrency: runtime.GOMAXPROCS(0),
+			Concurrency: 0,
```

Comment:
`0` znamena `auto` = pocet dostupnych CPU threads.

Teda ako default hodnotu dame `0`, a vyhodnotime to neskor.